### PR TITLE
Fix hourofcode.com/faq strings

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update the "Where can I print certificates for my students?" link on https://hourofcode.com/faq
- Created a new string

**Related PR:** https://github.com/code-dot-org/code-dot-org/pull/54106

**Jira ticket:** [ACQ-1050](https://codedotorg.atlassian.net/browse/ACQ-1050)

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 1]
```diff
@@ -1663,6 +1663,7 @@
   hoc_faq_keep_learning_a_markdown: "Anyone can host an Hour of Code at any time. The tutorials stay up year-round. You can expect all our tutorials and curriculum to be available on our site in perpetuity. Please go to [our resources](%{promote}) for event how-to guides and other resources to help make your Hour of Code event a success."
   hoc_faq_certificates_q: "Where can I print certificates for my students?"
   hoc_faq_certificates_a_markdown: "Go to our [certificates page](%{certificates_url}/certificates) where you can print certificates for your entire class ahead of time."
+  hoc_faq_certificates_answer: "Go to our [certificates page](%{certificates_url}) where you can print certificates for your entire class ahead of time."
```
Comment (Hamms) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: tiny nit: we do like to have "markdown" in the name of the key for strings that use markdown formatting. Something like 
```suggestion
  hoc_faq_certificates_answer_markdown: "Go to our [certificates page](%{certificates_url}) where you can print certificates for your entire class ahead of time."
```
would be ideal

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -1663,6 +1663,7 @@
   hoc_faq_keep_learning_a_markdown: "Anyone can host an Hour of Code at any time. The tutorials stay up year-round. You can expect all our tutorials and curriculum to be available on our site in perpetuity. Please go to [our resources](%{promote}) for event how-to guides and other resources to help make your Hour of Code event a success."
   hoc_faq_certificates_q: "Where can I print certificates for my students?"
   hoc_faq_certificates_a_markdown: "Go to our [certificates page](%{certificates_url}/certificates) where you can print certificates for your entire class ahead of time."
```
Comment (Hamms) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Is this string still being used anywhere after this change? If not, let's remove it as part of this PR!

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Updated 🙌

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Removed ❌

**Review by Hamms — APPROVED** [review]
LGTM; thank you!

**Pull request merged**